src/utils/api.ts

```diff
@@ -18,37 +18,43 @@ const getBaseUrl = () => {
   return `http://localhost:${process.env.PORT ?? 3000}`; // dev SSR should use localhost
 };
 
+const config = () => {
+  return {
+    /**
+     * Transformer used for data de-serialization from the server.
+     *
+     * @see https://trpc.io/docs/data-transformers
+     **/
+    transformer: superjson,
+
+    /**
+     * Links used to determine request flow from client to server.
+     *
+     * @see https://trpc.io/docs/links
+     * */
+    links: [
+      loggerLink({
+        enabled: (opts) =>
+          process.env.NODE_ENV === "development" ||
```

Comment:
zostało wspomniane aby nie używać process.env tylko robić import, tutaj chyba też się to aplikuje?